crates/cargo-contract/src/cmd/extrinsics/mod.rs

```diff
@@ -361,6 +361,14 @@ pub fn display_contract_exec_result_debug<R, const WIDTH: usize>(
     Ok(())
 }
 
+pub fn display_dry_run_result_warning() {
+    println!("You call {} been executed.", "has not".bold());
```

Comment:
```suggestion
    println!("Your call {} been executed.", "has not".bold());
```
Also I would not use the word `call` here, since it has an overloaded and specific meaning in this context. i.e. `instantiate`, `call`...

So you could pass the name of the command to this function or maybe use transaction instead.